src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingWorkCoordinator.UnitTestingAsyncWorkItemQueue.cs

```diff
@@ -157,8 +161,6 @@ public void Dispose()
                     RaiseCancellation_NoLock(cancellations);
                 }
 
-                protected Workspace Workspace => _workspace;
```

Comment:
goal was to get rid of storage of the workspace (and exposing it like this).  that way when this doesn't prevent us from removing RemoteWorkspace.